<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>secondWidget</class>
 <widget class="QWidget" name="secondWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MetroPlus</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/new/prefix1/image/logo.png</normaloff>:/new/prefix1/image/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#secondWidget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(248, 235, 226, 255), stop:0.5 rgba(250, 250, 250, 255), stop:1 rgba(250, 250, 250, 255));
}</string>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/LOGO2.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>100</y>
     <width>307</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border-image: url(:/new/prefix1/image/bg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="search">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
font: 11pt &quot;微软雅黑&quot;;
color: rgba(0,0,0,50);
background-color: rgba(0,0,0,0);
border-color: rgba(0,0,0,50);
border-width:1px 1px;
border-style:solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>  搜地点、搜线路</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 241, 241);
border-radius: 10px;
font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="startText">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;微软雅黑&quot;;
background:transparent;
border-width:0;
border-style:outset;</string>
   </property>
   <property name="text">
    <string>华中科技大学</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="endText">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-width:0;
border-style:outset;
font: 13pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>循礼门</string>
   </property>
  </widget>
  <widget class="QLabel" name="image1">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>129</y>
     <width>6</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/image1.png);
background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="changeButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>132</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/button1.png);
background-color: rgb(241, 241, 241);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="menuBg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>850</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(240, 240, 240,150);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>132</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/button2.png);
background:transparent;
border-width:0;
border-style:outset;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="menuWidget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>171</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>172</width>
      <height>263</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="menu">
     <item>
      <widget class="QPushButton" name="travel">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 9pt &quot;微软雅黑&quot;;
color: rgb(136, 136, 136);
background:transparent;
border-width:0;
border-style:outset;
text-align : left;</string>
       </property>
       <property name="text">
        <string>出行</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="1" column="1">
        <widget class="QPushButton" name="realTime">
         <property name="styleSheet">
          <string notr="true">QPushButton { 
background:transparent;
border-width:0;
border-style:outset;
font: 13pt &quot;微软雅黑&quot;;
color: rgba(0, 0, 0, 150);
 } 

QPushButton:hover { 
background-color: rgba(0,0,0,20);
border-radius: 8px;
}
</string>
         </property>
         <property name="text">
          <string>实时地铁     </string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/new/prefix1/image/realTimeIcon.png</normaloff>:/new/prefix1/image/realTimeIcon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="historyRoute">
         <property name="styleSheet">
          <string notr="true">QPushButton { 
background:transparent;
border-width:0;
border-style:outset;
font: 13pt &quot;微软雅黑&quot;;
color: rgba(0, 0, 0, 150);
 } 

QPushButton:hover { 
background-color: rgba(0,0,0,20);
border-radius: 8px;
}
</string>
         </property>
         <property name="text">
          <string>历史行程     </string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/new/prefix1/image/operationTimeIcon.png</normaloff>:/new/prefix1/image/operationTimeIcon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="operationTime">
         <property name="styleSheet">
          <string notr="true">QPushButton { 
background:transparent;
border-width:0;
border-style:outset;
font: 13pt &quot;微软雅黑&quot;;
color: rgba(0, 0, 0, 150);
 } 

QPushButton:hover { 
background-color: rgba(0,0,0,20);
border-radius: 8px;
}
</string>
         </property>
         <property name="text">
          <string>运营时间     </string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/new/prefix1/image/historyRouteIcon.png</normaloff>:/new/prefix1/image/historyRouteIcon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="routePlanning">
         <property name="styleSheet">
          <string notr="true">font: 13pt Microsoft YaHei UI;color: rgb(255, 255, 255);border-radius: 8px;background-color: rgba(0,0,0, 70);</string>
         </property>
         <property name="text">
          <string>路线规划     </string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/new/prefix1/image/routePlanningIcon.png</normaloff>:/new/prefix1/image/routePlanningIcon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="recommend">
       <property name="styleSheet">
        <string notr="true">font: 700 9pt &quot;微软雅黑&quot;;
color: rgb(136, 136, 136);
background:transparent;
border-width:0;
border-style:outset;
text-align : left;</string>
       </property>
       <property name="text">
        <string>推荐</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="1">
        <widget class="QPushButton" name="hotPlace">
         <property name="styleSheet">
          <string notr="true">QPushButton { 
background:transparent;
border-width:0;
border-style:outset;
font: 13pt &quot;微软雅黑&quot;;
color: rgba(0, 0, 0, 150);
 } 

QPushButton:hover { 
background-color: rgba(0,0,0,20);
border-radius: 8px;
}
</string>
         </property>
         <property name="text">
          <string>热门地点     </string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/new/prefix1/image/hotPlaceIcon.png</normaloff>:/new/prefix1/image/hotPlaceIcon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="person">
       <property name="styleSheet">
        <string notr="true">font: 700 9pt &quot;微软雅黑&quot;;
color: rgb(136, 136, 136);
background:transparent;
border-width:0;
border-style:outset;
text-align : left;</string>
       </property>
       <property name="text">
        <string>我的</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
font: 12pt &quot;黑体&quot;;
border-style:outset;
color: rgba(0, 0, 0, 150);</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
background:transparent;
font: 12pt &quot;黑体&quot;;
border-style:outset;
color: rgba(0, 0, 0, 150);
 } 

QPushButton:hover { 
color: rgba(0, 0, 0, 200);
}
</string>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/exit_image.png</normaloff>:/new/prefix1/image/exit_image.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="profile_photo">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-radius: 15px;
border-image: url(:/new/prefix1/image/head1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="cloth_button">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-style:outset;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/cloth_image.png</normaloff>:/new/prefix1/image/cloth_image.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>15</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>301</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,255,255);
border-radius: 12px;</string>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>271</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollArea{
background-color:transparent;
}</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>261</width>
       <height>800</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>800</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="shortestDst">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 11pt Microsoft YaHei UI;
color: rgb(0,0,0);
background: transparent;
}
QPushButton:checked{
font: 700 11pt Microsoft YaHei UI;
color: rgb(255, 255, 255);
border-radius: 12px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(65,88,132) stop:1 rgb(102,124,161));
}</string>
    </property>
    <property name="text">
     <string>距离最短</string>
    </property>
    <property name="iconSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="leastSta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 11pt Microsoft YaHei UI;
color: rgb(0,0,0);
background: transparent;
}
QPushButton:checked{
font: 700 11pt Microsoft YaHei UI;
color: rgb(255, 255, 255);
border-radius: 12px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(65,88,132) stop:1 rgb(102,124,161));
}</string>
    </property>
    <property name="text">
     <string>站点最少</string>
    </property>
    <property name="iconSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_map" native="true">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>210</y>
     <width>621</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <widget class="QLabel" name="mapLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>621</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/metromap.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>371</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,255,255);
border-radius: 10px;
padding-left: 10px;
padding-top: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="card1">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/card1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="card2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/card2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="card3">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/card3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="card4">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/card4.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="time_toHotplace">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>371</width>
     <height>131</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-width:0;
border-style:outset;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>widget_map</zorder>
  <zorder>widget</zorder>
  <zorder>menuBg</zorder>
  <zorder>logo</zorder>
  <zorder>label</zorder>
  <zorder>search</zorder>
  <zorder>label_2</zorder>
  <zorder>startText</zorder>
  <zorder>endText</zorder>
  <zorder>image1</zorder>
  <zorder>changeButton</zorder>
  <zorder>searchButton</zorder>
  <zorder>menuWidget</zorder>
  <zorder>username</zorder>
  <zorder>exit_button</zorder>
  <zorder>profile_photo</zorder>
  <zorder>cloth_button</zorder>
  <zorder>timeLabel</zorder>
  <zorder>card1</zorder>
  <zorder>card2</zorder>
  <zorder>card3</zorder>
  <zorder>card4</zorder>
  <zorder>time_toHotplace</zorder>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
